tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -20,6 +29,10 @@ def __init__(self, marionette):
             expected.element_displayed(*self._model_locator))
 
     @property
+    def screen(self):
```

Comment:
Wouldn't it be better to call this screen_element?
I mean, calling it .screen makes it sound like the Screen object or something like that.